Gems/MotionMatching/Code/Tests/MinMaxScalerTests.cpp

```diff
@@ -74,6 +74,63 @@ namespace EMotionFX::MotionMatching
         EXPECT_EQ(t(2,0), 1.0f); EXPECT_EQ(t(2, 1), 0.75f);EXPECT_EQ(t(2, 2), 0.25f);EXPECT_EQ(t(1, 3), 3.0f);
     }
 
+    TEST_F(MinMaxScalerFixture, TransformValueNoClipping)
+    {
+        FeatureMatrix m;
+        m.resize(2, 1);
+        m(0, 0) = -2.0f;
+        m(1, 0) = 2.0f; // range = 4.0
+
+        MinMaxScaler minMaxScaler;
+        EXPECT_TRUE(minMaxScaler.Fit(m));
+
+        EXPECT_EQ(minMaxScaler.Transform(-6.0f, 0), -1.0f);
+        EXPECT_EQ(minMaxScaler.Transform(4.0f, 0), 1.5f);
+    }
+
+    TEST_F(MinMaxScalerFixture, TransformValueClip)
+    {
+        FeatureMatrix m;
+        m.resize(2, 1);
+        m(0, 0) = -2.0f;
+        m(1, 0) = 2.0f;
+
+        MinMaxScaler minMaxScaler;
+        EXPECT_TRUE(minMaxScaler.Fit(m, {/*featureMin=*/0.0f, /*featureMax=*/1.0f, /*clip=*/true}));
+
+        EXPECT_EQ(minMaxScaler.Transform(-6.0f, 0), 0.0f);
+        EXPECT_EQ(minMaxScaler.Transform(8.0f, 0), 1.0f);
+    }
+
+    TEST_F(MinMaxScalerFixture, FeatureRangeTest)
```

Comment:
suggestion: This may be hard to do in all cases but it would be great to add slightly more descriptive test names to describe the behavior we expect from the code, and we can then see exactly what _isn't_ happening if the test fails. E.g. Something like... `ValueStaysInRangeAfterTransform`. This applies to a few of the tests, not critical but just makes things a lot easier for the uninitiated to understand a bit more quickly what's going on.